setup.py

```diff
@@ -48,11 +48,6 @@ def is_raspberry_pi(raise_on_errors=False):
     return True
 
 
-requires = [
```

Comment:
This is needed because during install, if we are on Raspberry Pi board then we add a dependency to Rpi.GPIO